sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomFormModel.cs

```diff
@@ -7,36 +7,96 @@
 namespace Azure.AI.FormRecognizer.Training
 {
     /// <summary>
+    /// Represents a custom model returned from training.
     /// </summary>
     public class CustomFormModel
     {
-        internal CustomFormModel()
+        internal CustomFormModel(Model_internal model)
         {
+            ModelId = model.ModelInfo.ModelId.ToString();
+            Status = model.ModelInfo.Status;
+            CreatedOn = model.ModelInfo.CreatedDateTime;
+            LastModified = model.ModelInfo.LastUpdatedDateTime;
+            Models = ConvertToSubmodel(model);
+            TrainingDocuments = model.TrainResult.TrainingDocuments;
         }
 
         /// <summary>
+        /// Model identifier.
         /// </summary>
         public string ModelId { get; }
 
         /// <summary>
+        /// The status of the model.
         /// </summary>
         public CustomFormModelStatus Status { get; }
 
         /// <summary>
+        /// Date and time (UTC) when the model was created.
         /// </summary>
-        public DateTime CreatedOn { get; }
+        public DateTimeOffset CreatedOn { get; }
 
         /// <summary>
+        /// Date and time (UTC) when the status was last updated.
         /// </summary>
-        public DateTime LastUpdatedOn { get; }
+        public DateTimeOffset LastModified { get; }
 
         /// <summary>
+        /// A list of submodels which contains the fields trained on.
         /// </summary>
         public IReadOnlyList<CustomFormSubModel> Models { get; }
 
         /// <summary>
+        ///  Training document information about the custom model.
         /// </summary>
         // TODO: for composed models, union what comes back on submodels into this single list.
         public IReadOnlyList<TrainingDocumentInfo> TrainingDocuments { get; }
+
+        private static IReadOnlyList<CustomFormSubModel> ConvertToSubmodel(Model_internal model)
```

Comment:
super tiny nit, but can we name this ConvertToSubmodel**s** ?

Reply:
updated